frontend/src/serviceWorkerRegistration.js

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable */
```

Comment:
This whole file is copied from https://github.com/cra-template/pwa/tree/main/packages/cra-template-pwa (which is linked from the CRA official docs re: PWA)